src/main/webapp/script.js

```diff
@@ -93,6 +95,21 @@ function getUserInfo() {
         document.getElementById('user-num-complete-container');
       numTaskCompleted.innerText =
         'Total Tasks Completed: ' + users[user].numTaskCompleted;
+      const priority_skills = document.getElementById('user-prskills-container');
+      pskills = '';
+      for (skill of users[user].skills){
+        if (skill.priority == true){
+          if(pskills == ''){
+            pskills = skill.skill;
+          }
+          else{
+            pskills = pskills + ', ' + skill.skill
+          }
+        }
+      }
+      console.log('pskills:');
+      console.log(pskills);
```

Comment:
pskills outputs in console what I want it to, yet the container shows [object,Object],[object,Object]. Any ideas why? I can't seem to figure this out...